files/templates/additional-scrape-configs.template.yaml

```diff
@@ -1875,4 +1875,23 @@
   - target_label: provider
     replacement: {{ .Provider }}
 {{ end }}
+- job_name: {{ .ClusterID }}-prometheus/vault-{{ .ClusterID }}/0
+  metrics_path: /metrics
+  scheme: http
+  static_configs:
+  - targets:
+    - {{ .Vault }}:10300
+    labels:
+      app: vault
+  - targets:
+    - {{ .Vault }}:9005
```

Comment:
What is this cert exporter about? Do we not have a cert exporter app?

Reply:
There's a cert-exporter app running on the cluster, but there's an additional cert-exporter on the vault node.